doc/api/addons.md

```diff
@@ -595,6 +595,8 @@ class MyObject : public node::ObjectWrap {
   static void PlusOne(const v8::FunctionCallbackInfo<v8::Value>& args);
   static v8::Persistent<v8::Function> constructor;
   double value_;
+  napi_env env_;
```

Comment:
The N-API code here are all unnecessary. When I mention the napi tests in https://github.com/nodejs/node/issues/19876#issuecomment-379524249 , I meant the tests demonstrates how it works *in the context of napi*.